docker/Dockerfile.prod

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.15
+
+SHELL ["/bin/bash", "-c"]
+
+# since we're starting non-interactive shell, 
+# we wil need to tell bash to load .bashrc manually
+ENV BASH_ENV ~/.bashrc
+# needed by volta() function
+ENV VOLTA_HOME /root/.volta
+# make sure packages managed by volta will be in PATH
+ENV PATH $VOLTA_HOME/bin:$PATH
+
+# Using volta to manage node + yarn versions
+RUN curl https://get.volta.sh | bash
+RUN volta install node
+
+WORKDIR /app
+COPY . /app
+
+RUN cd ui && yarn && yarn build-prod
+
+RUN go mod download
+RUN go build
+
+EXPOSE 8900
```

Comment:
I am pretty sure on prod we would wanna expose `8080` port. We will change the default port of the app using environment variables when app is deployed